compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -149,10 +158,13 @@ def __init__(
 
         # mypy ... sometimes you just ain't too bright
         self.container_id = container_id  # type: ignore[assignment]
+        self.user_endpoint_config = user_endpoint_config
 
         self._task_group_id: uuid.UUID | None = None  # help mypy out
         self.task_group_id = task_group_id
 
+        self.user_endpoint_config = user_endpoint_config
```

Comment:
Can the assignment on line 161 also be removed? It looks like we're doing the same thing twice, unless I'm missing something

Reply:
Ah, that's just a typo. Removing now.